packages/core/src/accessanalyzer/vue/iamPolicyChecks.vue

```diff
@@ -289,8 +322,25 @@ export default defineComponent({
         setCfnParameterFilePath: function (event: any) {
             this.initialData.cfnParameterPath = event.target.value
         },
-        runValidator: function () {},
-        runCustomPolicyCheck: function () {},
+        runValidator: function () {
+            client.validatePolicy(this.documentType, this.policyType, this.initialData.cfnParameterPath)
+        },
+        runCustomPolicyCheck: function () {
+            if (this.checkType == 'CheckNoNewAccess') {
+                client.checkNoNewAccess(
+                    this.documentType,
+                    this.resourcePolicyType,
+                    this.initialData.referenceDocument,
+                    this.initialData.cfnParameterPath
+                )
+            } else {
```

Comment:
i'd not assume there is only two checks

Reply:
Yup, I agree with this. We should explicitly have `else if` for checking with other API